engine/schema/src/main/resources/META-INF/db/schema-41400to41500.sql

```diff
@@ -18,3 +18,12 @@
 --;
 -- Schema upgrade from 4.14.0.0 to 4.15.0.0
 --;
+
+-- Change guest OS name to support default CentOS 5 template in XenServer8.0
+UPDATE `cloud`.`guest_os_hypervisor` SET guest_os_name='CentOS 7' where guest_os_id=(SELECT guest_os_id from `cloud`.`vm_template` WHERE unique_name='centos56-x86_64-xen') AND hypervisor_type='Xenserver' AND hypervisor_version='8.0.0';
```

Comment:
@rhtyd do I need to add this to support default CentOS template or will this just be a workaround at the runtime?

Reply:
workaround is fine for now, but I think the default built-in template does not work with this right?